packages/scripts/src/application.ts

```diff
@@ -130,10 +163,19 @@ export class Application {
             publicPath,
             title,
             configurations,
-            staticBuild: true,
+            staticBuild,
             publicConfigsRoute,
             overrideConfig,
             singleFeature,
+            isExternal: external,
+            externalFeatures:
```

Comment:
isExternal === should build this feature in external mode
externalFeatures === should add the external features to the built output